### PR TITLE
Update get_package_properties.py logic for python 2.7

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update `eng/scripts/get_package_properties.py` logic to enable it work for python 2.7

### CODE REVIEW

File: eng/scripts/get_package_properties.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
     parser.add_argument('-s', '--search_path', required=True, help='The scope of the search')
     args = parser.parse_args()
 
-    for p in glob.glob(args.search_path, recursive=True):
```
Comment (weshaggard) [inline eng/scripts/get_package_properties.py:14]: What about this doesn't work for python 2.7?

Comment (weshaggard) [reply eng/scripts/get_package_properties.py:14]: Also why do we care about 2.7 support in this script? Is there a case we are running in that context?

Comment (chidozieononiwu) [reply eng/scripts/get_package_properties.py:14]: `glob.glob( ,recursive=True)` does not work in python 2.7. This causes it to fail when it is run on some test pipelines that are using python 2.7

**Review by weshaggard — APPROVED** [review]
The changes look reasonable to me.

**Pull request merged**